packages/atoms/src/lib/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { type ClassValue, clsx } from "clsx";
+import { twMerge } from "tailwind-merge";
+
+export function cn(...inputs: ClassValue[]) {
```

Comment:
Personally, i would call this function `classNames` as it's more descriptive, but others might prefer the shorthand 🤷

Reply:
Also, can we rename `lib` folder to just `utils` and then name the file `classNames.ts` or something?

If we have `lib/utils.ts` file then it will become a place where people just dump all utils together and after a while it will be hard to understand what kind of utils exactly are there.

Reply:
The classnames and folders are being generated hence I thought not to rename or tweak anything since it'll be uniform that way  for all the components now and also in the future

Reply:
Gotcha, what code generates them?

Reply:
`npx shadcn-ui@latest add <component_name>` generated an src folder in the root https://github.com/calcom/cal.com/tree/connect-component/packages/atoms/src which contains the lib and components folder for all the shadcn components